src/main/java/frc/robot/path/AutoPathBuilder.java

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
i think i love this class. 
the only reason i dont know i love it right off is because i'd like to see a test case that uses it so i can see how it looks.